apps/web/src/app/vault/org-vault/vault.component.ts

```diff
@@ -612,6 +619,16 @@ export class VaultComponent implements OnInit, OnDestroy {
       }),
     );
 
+    this.resellerWarning$ = combineLatest([organization$]).pipe(
```

Comment:
I think we only need `combineLatest` when we have multiple independent streams. This can be simplified a bit

```suggestion
    this.resellerWarning$ = organization$.pipe(
    filter(org => org.isOwner),
    switchMap(org => this.billingApiService
      .getOrganizationBillingMetadata(org.id)
      .pipe(map(metadata => ({org, metadata})))
    ),
    map()
```